src/wren_compiler.c

```diff
@@ -1381,17 +1381,20 @@ typedef enum
 {
   PREC_NONE,
   PREC_LOWEST,
-  PREC_ASSIGNMENT, // =
-  PREC_LOGIC,      // && ||
-  PREC_EQUALITY,   // == !=
-  PREC_IS,         // is
-  PREC_COMPARISON, // < > <= >=
-  PREC_RANGE,      // .. ...
-  PREC_BITWISE,    // | ^ & << >>
-  PREC_TERM,       // + -
-  PREC_FACTOR,     // * / %
-  PREC_UNARY,      // unary - ! ~
-  PREC_CALL,       // . () []
+  PREC_ASSIGNMENT,    // =
+  PREC_LOGIC,         // && ||
```

Comment:
We should probably split these out too. C makes `&&` higher precedence than `||`.